src/pages/workspace/reportFields/WorkspaceListValuesPage.tsx

```diff
@@ -0,0 +1,271 @@
+            options.push({
+                icon: Expensicons.Trashcan,
+                text: translate(selectedValuesArray.length === 1 ? 'workspace.reportFields.deleteValue' : 'workspace.reportFields.deleteValues'),
+                value: CONST.POLICY.REPORT_FIELDS_VALUES_BULK_ACTION_TYPES.DELETE,
+                onSelected: () => setDeleteValuesConfirmModalVisible(true),
+            });
+
+            const enabledValues = selectedValuesArray.filter((valueName) => {
+                const index = formDraft?.listValues?.indexOf(valueName) ?? -1;
+                return !formDraft?.disabledListValues?.[index];
+            });
+
+            if (enabledValues.length > 0) {
+                const valuesToDisable = selectedValuesArray.reduce<number[]>((acc, valueName) => {
+                    const index = formDraft?.listValues?.indexOf(valueName) ?? -1;
+                    if (!formDraft?.disabledListValues?.[index] && index !== -1) {
+                        acc.push(index);
+                    }
+
+                    return acc;
+                }, []);
+
+                options.push({
+                    icon: Expensicons.DocumentSlash,
+                    text: translate(enabledValues.length === 1 ? 'workspace.reportFields.disableValue' : 'workspace.reportFields.disableValues'),
+                    value: CONST.POLICY.REPORT_FIELDS_VALUES_BULK_ACTION_TYPES.DISABLE,
+                    onSelected: () => {
+                        setSelectedValues({});
+                        ReportFields.setReportFieldsListValueEnabled(valuesToDisable, false);
+                    },
+                });
+            }
+
+            const disabledValues = selectedValuesArray.filter((valueName) => {
+                const index = formDraft?.listValues?.indexOf(valueName) ?? -1;
+                return formDraft?.disabledListValues?.[index];
+            });
+
+            if (disabledValues.length > 0) {
+                const valuesToEnable = selectedValuesArray.reduce<number[]>((acc, valueName) => {
+                    const index = formDraft?.listValues?.indexOf(valueName) ?? -1;
+                    if (formDraft?.disabledListValues?.[index] && index !== -1) {
+                        acc.push(index);
+                    }
+
+                    return acc;
+                }, []);
+
+                options.push({
+                    icon: Expensicons.Document,
+                    text: translate(disabledValues.length === 1 ? 'workspace.reportFields.enableValue' : 'workspace.reportFields.enableValues'),
+                    value: CONST.POLICY.REPORT_FIELDS_VALUES_BULK_ACTION_TYPES.ENABLE,
+                    onSelected: () => {
+                        setSelectedValues({});
+                        ReportFields.setReportFieldsListValueEnabled(valuesToEnable, true);
+                    },
+                });
+            }
+
+            return (
+                <ButtonWithDropdownMenu
+                    onPress={() => null}
```

Comment:
Is this intended?

Reply:
Yes, because I don't need any additional actions. I have handlers for dropdown buttons only. It is a common pattern for this component in the app as well.


https://github.com/Expensify/App/assets/57314631/a3d8fa5a-338d-4333-adec-fab4e661e3bc